setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup, find_packages
 
 EXTRAS_REQUIRE = {
-    "reco": ["python-dateutil>=2.7.0", "simplejson"],
+    "reco": ["simplejson"],
```

Comment:
We can probably just remove `"reco"` full stop. `simplejson` is only used in the tests, so you can just move it to the line below.